umm-lib/src/cmr/umm/iso_smap/granule/orbit.clj

```diff
@@ -115,17 +116,17 @@
   [field value]
   (try
     (date-time-parser/parse-datetime value)
-    (catch Exception e
+    (catch Exception _e
       (info (format "For Orbit calculated spatial domain field [%s] the value [%s] is not a datetime." field value))
       value)))
 
-(defn- parse-double
+(defn- oribt-parse-double
```

Comment:
Looks like you might have a typo here. oribt-parse-double -> orbit-parse-double

Reply:
Yeah, I do.  Thanks. It is now fixed.